### PR TITLE
#171380741 Setup API documentation

### PR DESCRIPTION
**Pull request by the22mastermind**
#### What does this PR do?
Setup API documentation using Swagger
#### Description of Task to be completed?
* Initialize an express server in index.js file
* Create a basic swagger.json spec file
* Install `express`, `swagger-ui-express` library
* Create a sample route at `/api`
#### How should this be manually tested?
1. Open your terminal and run `git clone https://github.com/Stackup-Rwanda/stackup2-barefoot-backend/`
2. run `cd stackup2-barefoot-backend` to go into the root directory of the project
3. run `git checkout -b ch-setup-api-documentation-171380741 origin/ch-setup-api-documentation-171380741` to pull this branch and switch to it
4. run `npm install` to install the dependencies
5. run `npm run start` to start the server
6. Open your browser and navigate to this link: [localhost:3000/public/api-docs](http://localhost:3000/public/api-docs)
7. Click on the highlighted `GET` route, then click on the `Try it out` button then `execute` button and you should see a server response with a status of `200` and a message of `welcome` 
* When this PR is merged, visit the link at step 6 above [here](https://stackup2-bar-ch-setup-a-ouacfx.herokuapp.com/public/api-docs) 
#### Any background context you want to provide?
* You need to install [Node.js](https://nodejs.org/en/) and [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) before testing.
#### What are the relevant pivotal tracker stories?
[#171380741](https://www.pivotaltracker.com/story/show/171380741)
#### Screenshots (if appropriate)
Barefoot Nomad API Doc 1
Barefoot Nomad API Doc 2

#### Questions:

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,93 +1,24 @@
...
-}
-
-require("./models/User");
+const router = express.Router();
 
+router.use('/public/api-docs', serve, setup(SwaggerSpecs));
 
+app.use(router);
 
-app.get('/', (req, res) => {
```
Comment (descholar-ceo) [inline src/index.js]: I think these codes `line 52-3` shouldn't have deleted, because this is making tests to fail, and is affecting your PR's deployment

**Review by descholar-ceo — CHANGES_REQUESTED** [review]

File: src/index.js (hunk 1 above)
Comment (the22mastermind) [reply src/index.js]: Alright. Let me fix it. Thanks for the feedback.

**Review by descholar-ceo — APPROVED** [review]
LGTM

**Review by ricahcyuzuzo — APPROVED** [review]

File: src/index.js [hunk 2]
```diff
@@ -17,6 +25,12 @@ app.get('/', (req, res) => {
   res.status(200).json({ message: 'welcome' });
 });
 
+app.get('/api', (req, res) => {
```
Comment (mifeille) [inline src/index.js]: Please remove this endpoint, it is decreasing our coverage and we already have that endpoint.

Comment (the22mastermind) [reply src/index.js]: Noted. Thanks for the feedback.

**Pull request merged**